modules/ROOT/pages/troubleshooting-guest-users-not-seeing-published-api-versions.adoc

```diff
@@ -0,0 +1,12 @@
+= Guest Users Not Seeing Published API Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+
+If guest users do not see published API versions that have been set to be visible for everyone:
```

Comment:
I just realized this, but we do have a template for troubleshooting . 

You could do something like

_Ensure guest users can se published API versions that have been set to be visible for everyone.  To do so:_


Or

_Scenario: Guest users do not see published API versions that have been set to be visible for everyone. To correct this issue_


I would consider this change throughout